### DISCUSSION TITLE
Extended CLI functionality to include all remote services provided by bimmer_connected.remote_services

### DISCUSSION THREAD

**Discussion by rsteininger** [bimmerconnected/bimmer_connected#411] [Feature request]
hi there, 
As I am using node red with a function call to the bimmer_connected CLI, I have added all (?) functions provided by the bimmer_connected.remote_services. 

As I am not too familiar with formal github pull request / branching, maybe this contribution could be included in the formal repository by someone more familiar / glued in with such a request? 

I have "tried" the CLI commands with our 2016 BMW i3 60Ahs. They were all reported "EXECUTED", but not all services did trigger the respective functions in our car ... not sure if that is because of an "error" in my programming or if the car simply does not support all the functions ... 

**CLI documentation would need to be changed to reflect the "bimmerconnected" function call:**
[https://bimmer-connected.readthedocs.io/en/latest/cli.html#bimmerconnected](url)
{status,fingerprint,lightflash,chargenow,remoteairconditioning,remoteairconditioningstop,remotedoorlock,remotedoorunlock,remotehornvehiclefinder,image,sendpoi,sendpoi_from_address,sendmessage}

                       ...

**Same for "Positional Arguments"**
[https://bimmer-connected.readthedocs.io/en/latest/cli.html#Positional%20Arguments](url)
Possible choices: status, fingerprint, lightflash, chargenow,remoteairconditioning,remoteairconditioningstop,remotedoorlock,remotedoorunlock,remotehornvehiclefinder, image, sendpoi, sendpoi_from_address, sendmessage

the update program: cli.py

Sorry, I tried to paste the code here but it seems to have formatting that reverts the "code paste" function of this post to text ... see below for the text file / python program file!

cli file to be renamed to .py:
[cli.py.txt](https://github.com/bimmerconnected/bimmer_connected/files/7870107/cli.py.txt)

Many thanks for your consideration!

**Comment by rikroe** [comment]
I'll have a look. Maybe there is also a way to make it more generic, i.e. `bimmerconnected remote_service {name_of_service} ...` that gets automatically filled by the available services.
